README.md

```diff
@@ -89,174 +127,120 @@ pipeline.execute() // execute the pipeline
 // are loaded in this case into JVM memory as a single Array[Byte]
 var pointCloud = pv.getPointCloud();
 var x = pointCloud.getDouble(0, DimType.X()); // get a point with PointId = 0 and only a single dimensions
-
 // in some cases it is not neccesary to load everything into JVM memory
 // so it is possible to get only required points directly from the PointView
 var y = pv.getDouble(0, DimType.Y());
-
 // it is also possible to get access to the triangular mesh generated via PDAL
 var mesh = pv.getTriangularMesh();
 // the output is an Array of Triangles
 // Each Triangle contains PointIds from the PDAL point table
 var triangles = mesh.asArray();
-
 pv.close();
 pvs.close();
 pipeline.close();
 ```
-
 ### PDAL Scala
-
 ```scala
 import io.pdal._
 import io.pdal.pipeline._
-
 // To construct the expected json
 val expected =
-  """
-     |{
-     |  "pipeline" : [
-     |    {
-     |      "filename" : "/path/to/las",
-     |      "type" : "readers.las"
-     |    },
-     |    {
-     |      "type" : "filters.crop"
-     |    },
-     |    {
-     |      "filename" : "/path/to/new/las",
-     |      "type" : "writers.las"
-     |    }
-     |  ]
-     |}
-  """.stripMargin
-  
+"""
+|{
+|  "pipeline" : [
+|    {
+|      "filename" : "/path/to/las",
+|      "type" : "readers.las"
+|    },
+|    {
+|      "type" : "filters.crop"
+|    },
+|    {
+|      "filename" : "/path/to/new/las",
+|      "type" : "writers.las"
+|    }
+|  ]
+|}
+""".stripMargin
 // The same, but using scala DSL
 val pc = ReadLas("/path/to/las") ~ FilterCrop() ~ WriteLas("/path/to/new/las")
-
 // The same, but using RawExpr, to support not implemented PDAL Pipeline API features
 // RawExpr accepts a circe.Json type, which can be a json object of any desired complexity
-val pcWithRawExpr = ReadLas("/path/to/las") ~ RawExpr(Map("type" -> "filters.crop").asJson) ~ WriteLas("/path/to/new/las") 
-
+val pcWithRawExpr = ReadLas("/path/to/las") ~ RawExpr(Map("type" -> "filters.crop").asJson) ~ WriteLas("/path/to/new/las")
 // Create Pipelines from the constructed expressions
 val pipelinePc = pc.toPipeline
 val pipelinePc = pcWithRawExpr.toPipline
 ```
 
-## How to compile
-
-Development purposes (including binaries) compilation:
-  1. Install PDAL (using brew / package managers (unix) / build from sources / etc)     
-  2. Build native libs `./sbt native/nativeCompile` (optionally, binaries would be built during tests run)
-  3. Run `./sbt core/test` to run PDAL tests
-
-Only Java development purposes compilation:
-  1. Provide `$LD_LIBRARY_PATH` or `$DYLD_LIBRARY_PATH`
-  2. If you don't want to provide global variable you can pass `-Djava.library.path=<path>` into sbt:
-    `./sbt -Djava.library.path=<path>`
-  3. Set `PDAL_DEPEND_ON_NATIVE=false` (to disable `native` project build)
-  4. Run `PDAL_DEPEND_ON_NATIVE=false ./sbt`
-
-Finally the possible command to launch and build PDAL JNI bindings could be:
-
-```bash
-# Including binaries build
-./sbt
+## Known Issues
```

Comment:
I won't say it's a known issue, I like the previous title more; it's like a possible issue, i.e. I don't know how to constantly reporduce such env state.